sdk/servicebus/Azure.Messaging.ServiceBus/src/Authorization/SharedAccessCredential.cs

```diff
@@ -129,17 +128,17 @@ public SharedAccessCredential(AzureSasCredential sourceCredential)
                 if ((!string.Equals(signature.SharedAccessKeyName, name, StringComparison.Ordinal))
```

Comment:
Is the null check for _sourceKeyCredential on line 124 necessary?

Reply:
Yes;  when we create this from the connection string, there is no named key credential.  The shared key/value cannot be changed, only the expiration extended.